src/index.html

```diff
@@ -3,10 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Resturant</title>
 </head>
 <body>
-    <p class="text-danger">asjdkajsdh</p>
-    <img src="img/b.png" alt="as">
+    <main></main>
```

Comment:
![image](https://user-images.githubusercontent.com/31897434/86682584-a2106700-bff8-11ea-8dcb-43fa86009d58.png)


In other words;
You are supposed to have `<div id="content"></div>` in here.